lib/matplotlib/bezier.py

```diff
@@ -316,7 +317,7 @@ def _f(xy):
 
 def get_cos_sin(x0, y0, x1, y1):
     dx, dy = x1 - x0, y1 - y0
-    d = (dx * dx + dy * dy) ** .5
+    d = (dx * dx + dy * dy) ** .5 or 1  # Account for divide by zero
```

Comment:
Like this?
```python
L = np.hypot(dx, dy)
# Account for divide by zero
if L == 0:
    L = 1
```
This would also need to be changed in `Curve(_Base)` and `FancyArrow(Polygon)`, for consistency.